sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -246,6 +251,9 @@ export interface ReceivedMessageWithLock extends ReceivedMessage {
 export interface ReceiveMessagesOptions extends OperationOptionsBase, WaitTimeOptions {
 }
 
+// @public
+export type ReceiveModes = "peekLock" | "receiveAndDelete";
```

Comment:
What is the reason for this to be plural?

Reply:
Hmmm.. changing this to `ReceiveMode`.